java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/LspTemplateUI.java

```diff
@@ -165,8 +173,20 @@ private CompletableFuture<Object> projectUI(DataFolder templates, NbCodeLanguage
     }
 
     private static CompletionStage<String> findPackage(CompletionStage<?> uiBefore, NbCodeLanguageClient client) {
-        return uiBefore.thenCompose((__) ->
-            client.showInputBox(new ShowInputBoxParams(Bundle.CTL_TemplateUI_SelectPackageName(), Bundle.CTL_TemplateUI_SelectPackageNameSuggestion()))
+        return uiBefore.thenCompose((__) -> {
+            class ValidatePackageName implements Function<String, CompletionStage<String>> {
+
+                @Override
+                public CompletionStage<String> apply(String packageName) {
+                    if (!SourceVersion.isName(packageName)) {
```

Comment:
Here, we are specifically looking for a package name, so the check makes sense to me.